### PR TITLE
3.0 pr template

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,6 +48,7 @@ ZK_JAR=$(ZK_JAR_PATH)/$(ZK_JAR_NAME)
 SHELL = /bin/bash
 
 prepareLic:
+	echo 'This makefile is for ci test and have dependency, do not run locally, if you want to run unit test, please run `go test ./...` directly'
```
Comment (AlexStocks) [inline Makefile:51]: echo 'The makefile is for ci test and has dependencies. Do not run locally. If you want to run the unit tests, run command`go test ./...` directly'

File: Makefile [hunk 2]
```diff
@@ -15,6 +15,10 @@
 # limitations under the License.
 #
 
+# DO NOT RUN
+# This makefile is for ci test and have dependency, do not run locally
```
Comment (AlexStocks) [inline Makefile:19]: # The makefile is for commit test and has dependencies. Do not run locally. 
# If you want to run the unit tests, run command `go test ./...` directly.

File: Makefile (hunk 1 above)
Comment (LaurenceLiZhixin) [reply Makefile:51]: The suggestion from Mr.Yu is reasonable, and I'll fix it right away.

Comment (AlexStocks) [reply Makefile:51]: Mr.Yu? haha.

**Review by zhaoyunxing92 — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by Mulavar — APPROVED** [review]

**Pull request merged**